store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java

```diff
@@ -84,7 +84,7 @@
     @ImportantField
     private int fileReservedTime = 72;
     // Flow control for ConsumeQueue
-    private int putMsgIndexHightWater = 600000;
```

Comment:
This field should be addressed in our doc. You know, if we do not have any comments, people would feel a  little quirk in these points.

Reply:
Could you tell me which doc this field should be addressed？ I just change this field name，I could not find the field named "putMsgIndexHightWater" in any doc under the path rocketmq/docs. thx

Reply:
@ni-ze please help to review this pr

Reply:
@HScarb thx for the answer

Reply:
@Hen1ng Fields in `MessageStoreConfig` may be set by reflection (you may find such usage by `org.apache.rocketmq.common.MixAll#properties2Object`), once you change the field names, users who have configured by the previous names will be affected, the values they've changed in property file will be 'restored' to default values. So changing these fields require careful consideration.

IMO, you'd better move this modification to another dedicated issue, and change the field name in a compatible way also with clear documents  addressing to it.